eng/Analyzers.props

```diff
@@ -1,5 +1,5 @@
 <Project>
-  <ItemGroup>
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
```

Comment:
You don't need the condition here as you already have it for the .depproj import. I would move it back to the Tools.props file.

Reply:
I rather have the condition in just one place, so removing it from the .depproj import and leaving it here, does that make sense?

Reply:
I prefer to not import a file which isn't doing anything.

Reply:
Yeah, I'm kind of on the middle ground for that, cause having multiple conditions is more error prone to someone mess up the condition, but since this is temporary I'm OK with either.